common/make/executable_module_recursive_build.mk

```diff
@@ -12,31 +12,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
-#
-# This file contains some helper definitions for including the Chrome Native
-# Client modules in Chrome Apps.
-# TODO(#177): Generalize the file to support Emscripten as well.
+# This file contains provides helper definitions for performing recursive make
```

Comment:
"contains provides"...this is probably a typo?

Reply:
Indeed. Done.